packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.jsx

```diff
@@ -38,8 +42,15 @@ const LowBalanceMessage = () => (
 );
 
 const BuySellModalFooter = ({ onCancel, is_submit_disabled, onSubmit }) => {
+    const { my_profile_store } = useStores();
     return (
-        <React.Fragment>
+        <div
+            className={
```

Comment:
could you use classNames()

Reply:
why do we need classNames here?

Reply:
Do we need it here actually? or is this fine using ternary here

Reply:
if not, I think its fine